tests/unittests/DeviceManagerTest.cpp

```diff
@@ -481,6 +482,18 @@ TEST_P(DeviceManagerTest, ReuseModule) {
   EXPECT_TRUE(result2->isEqual(output2));
 }
 
+TEST(Runtimetest, SetDeviceSize) {
```

Comment:
Every other test case in this file is part of `DeviceManagerTest`, mind using that instead of `Runtimetest`?